### PR TITLE
Checkpoint 2 Defense review

### PR DESCRIPTION
**Pull request by megwali**

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -47,21 +42,7 @@ def update
 
   def destroy
     @booking.destroy
-    redirect_to bookings_url, notice: "Booking was successfully Cancelled"
-  end
-
-  def past
-    @bookings = Booking.where(user_id: session[:user_id])
```
Comment (megwali) [inline app/controllers/bookings_controller.rb:54]: Moved to Bookings model

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -47,21 +42,7 @@ def update
...
-    render "index"
-  end
-
-  def search_booking
```
Comment (megwali) [inline app/controllers/bookings_controller.rb:58]: Moved to manage_bookings_controller

File: app/controllers/bookings_controller.rb [hunk 3]
```diff
@@ -84,69 +65,32 @@ def booking_params
   end
 
   def set_flight_select
-    session[:passenger_enquiry]["Flight Selected"] = params[:select_flight] if
-      params[:select_flight]
-    @passenger_enquiry = session[:passenger_enquiry]
-    if @passenger_enquiry && @passenger_enquiry["Flight Selected"]
-      @flight_selected = Flight.find(@passenger_enquiry["Flight Selected"])
-    elsif @booking
-      @flight_selected = Flight.find(@booking.flight_id)
```
Comment (megwali) [inline app/controllers/bookings_controller.rb:93]: Refactored

File: app/controllers/flights_controller.rb [hunk 4]
```diff
@@ -50,10 +51,10 @@ def flight_params
     params.require(:flight).permit(:origin,
                                    :destination,
                                    :seat,
-                                   :flight_cost,
```
Comment (megwali) [inline app/controllers/flights_controller.rb:53]: Adjusted naming

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -41,21 +37,7 @@ def update
...
-
-  def attempt_login
-    check_user
-    authorize_the_user
-  end
-
-  def logout
-    session[:user_id] = nil
-    session[:user_username] = nil
-    redirect_to login_path, notice: "User successfully signed out."
```
Comment (megwali) [inline app/controllers/users_controller.rb:58]: Moved to sessions controller

File: app/views/bookings/search_booking.html.erb [hunk 6]
```diff
@@ -1,8 +0,0 @@
-<h1>Find Your Booking Information</h1>
-<div class="form-group">
-<%= form_for(:booking, :url => {:action => "search_booking"}, :method => "get") do |seek| %>
-  <%= label_tag :reference_id, "Enter Your Booking Reference Number" %><br>
-  <%= text_field_tag :reference_id %>
-  <%= seek.submit "Get Booking Details"%>
-<% end %>
-</div>
```
Comment (andela-megwali) [inline app/views/bookings/search_booking.html.erb]: Moved to manage bookings view

**Pull request merged**